### DISCUSSION TITLE
🚨 WalletConnect V1 Shutdown🚨

### DISCUSSION THREAD

**Discussion by crypblizz8** [WalletConnect/walletconnect-monorepo#2810] [Announcements]
WalletConnect V1 is shutting down on 28 June 2023 in favour of WalletConnect V2.

Read our latest blogpost on what this means for you as a user or developer via our [Medium post](https://medium.com/walletconnect/walletconnect-v1-0-shutdown-what-to-expect-and-how-to-prepare-2ae443d6d7bd).

If you are a wallet/dapp developer and looking for more technical information, read our [Migration Guide here](https://docs.walletconnect.com/2.0/advanced/migration-from-v1.x/overview)

If you have any questions or issues, please provide the following information:

If you are a `dapp`:
1. The version of the package you are using
2. Reproducible code via a codeBox
3. The wallet (platform and version) you are using to get to this error
4. Whether you get errors on https://react-web3wallet.vercel.app/
5. How many users are impacted by this (estimates are fine)
6. How would you rate the severity of this (P0 — P1 — P2). P0 being it affects all WalletConnect users. P1 affecting several applications. P2 smaller severity.

If you are a `wallet`:

1. The version of the SDK you are using
2. Provide the reproduction path for this issue
3. The wallet (platform - iOS or Android and version) you are using to get to this error
4. Provide a dapp that you test against (if necessary)
5. How many users are impacted by this (estimates are fine)
6. How would you rate the severity of this (P0 — P1 — P2). P0 being it affects all WalletConnect users. P1 affecting several applications. P2 smaller severity.

And make a post at our Github Discussion here under [v1-v2-migration-support](https://github.com/orgs/WalletConnect/discussions/categories/v1-v2-migration-support).

**Comment by Nakamoto-dotcom** [comment]
Innovation 👏

**Comment by outlawcode** [comment]
index.js:82 WebSocket connection to 'wss://z.bridge.walletconnect.org/?env=browser&host=localhost%3A6091&protocol=wc&version=1' failed: 
_socketCreate @ index.js:82
eval @ index.js:92

**Reply by crypblizz8** [reply]
Please create a discussion as this is an announcement
